src/antelope/stores/user.ts

```diff
@@ -79,6 +102,34 @@ export const useUserStore = defineStore(store_name, {
             } finally {
                 useFeedbackStore().unsetLoading('account.loadUsers');
             }
+
+            this.loadCurrencyPreferences();
+        },
+        loadCurrencyPreferences() {
+            this.trace('loadCurrencyPreferences');
+            let shouldSetDefaultPreferences = false;
+
+            try {
+                const { locale, currency } = JSON.parse(localStorage.getItem(CURRENCY_PREFERENCES_KEY) || '{}');
+
+                if (![typeof locale, typeof currency].every(type => type === 'string')) {
```

Comment:
I would suggest adding a comment on this part because is kinda difficult to understand at first read.

```suggestion

                // if any locale or currency is not a string set defaults
                if (![typeof locale, typeof currency].every(type => type === 'string')) {
```